### ISSUE TITLE
EXC_BAD_ACCESS 

### ISSUE THREAD

**Issue by MussaCharles** [malcommac/SwiftDate#739]
I am getting ```EXC_BAD_ACCESS ``` when running the app from Xcode to a device. This exception is being triggered in ```RelativeFormatter``` class  when an instance of of language table is being cached on a method 
```swift 
private func language(forLocale locale: Locale) -> RelativeFormatterLang
```

Line 176: 
```swift
languagesCache[localeId] = instanceOfTable // Thread 17: EXC_BAD_ACCESS (code=1, address=0x8000000000000008)
```

![Screen Shot 2020-07-08 at 4 21 28 PM](https://user-images.githubusercontent.com/25889447/86889708-249f4080-c137-11ea-85c7-dd794c24b2da.png)

Any thoughts on this?

**Comment by alexiscn** [comment]
It seems that function should be called from Main Thread. I have the same issue when using SwiftDate with AsyncDisplayKit. 
After change format time to main thread. the crash gone.

**Comment by MussaCharles** [comment]
Hi, @alexiscn  Thanks for the workaround suggestion, I will try it once I get back to the project.

**Comment by malcommac** [comment]
I've fixed it for 6.2.0 and access is atomic right now.
